### DISCUSSION TITLE
Yağma yapan yol kesenlerin konumlarını bildiren uygulama

### DISCUSSION THREAD

**Discussion by yasin03** [acikkaynak/deprem-yardim-projesi#30] [Ideas]
Yağma yapan ve yol kesen kişilerin konumlarını ve yüz fotolarını ihbar olarak uygulamada gösteren güvenlik güçlerine lokasyon bazlı ve istatiksel olarak bilgi sunan bir uygulama fikri aklıma geldi sizce nasıl fikir mantıklımı?

**Comment by berkAktug** [comment]
Olabilecek bir şey aslında ama kötüye kullanımı olabilecek bir yönü var. İfşa kültürü tehlikeli bir kültür.
Alternatif olarak konum bilgilerini deprem.io üzerinde bulunan haritaları kullanabiliriz/entegre edilebilir.

**Comment by IbrahimbgrOzcan** [comment]
Yada yardım için yola çıkan tırları gps üzerinden takip edilebilecek site üzerinden anlık olarak izleme eklenebilir yardım çıktığı anda şöförlerin telefonlarına yüklenecek teyitli bi şekilde tabi bi ekip sadece bunu takip edebilir herhangi ani bi yağma durumunda geri bildirim ile cimere yada emniyet genel müdürlüğüne anlık konum şoför ismi gidip anında etkileşim sağlanabilir

**Reply by sercangoger** [reply]
hocam çok söyledim halaen söylüyorum. şuan ki deprem.io için gösterilebilir. 
herkes görmesine gerek yok. sistemi sadece AFAD ve ahbap yetkililerine açabilirler bu sayede. daha güvenli olacaktır.

**Reply by mahsumurebe** [reply]
tırların konumlarını GPS ile göstermek demek, yağmacıların ekmeğine yağ sürmek demek. sistemden hangi tırın nerde olduğunu takip edip eliyle koymuş gibi bulmasına yarayacaktır. @sercangoger e katılıyorum. böyle bir sistemin güvenliğinin üst düzeyde sağlanması sartı ile, kolluk kuvvetleri ve yetkili kuruluşlar ile paylaşımı daha doğru olur.

**Comment by ahmetcumhurarslan** [comment]
- tır şöförlerinin yükleyeceği program ile anlık koordinatlarının bir merkezde toplanıp güzergahtan olağan dışı çıkışları analiz edip olası yağma ihtimalleri değerlendirilebilir.
- tırlar için açık kapalı yolların haritaları geçmiş veriler kullanılarak çıkarılabilir.
- şöförlerin daha önce aynı bölgeye giden diğer şöförlerle irtibata geçebilmesi sağlanarak bölge şartları hakkında bilgi almaları sağlanabilir.
- aynı uygulamada tek tuşla yağma bildirimi yapılabilir. böylece anlık koordinat güvenlik güçleriyle paylaşılabilir.
- erzakların dağıtılması için lokal gönüllü organizasyonları gerçekleştirilip, şöförlerin bu insanlarla direkt iletişim kurmaları sağlanabilir. bahsettiğiniz gibi burda önemli nokta tırın datasının sadece sistemde olup dışarda görünmemesidir.

aklıma gelen bir diğer konu, gönderilen erzak kutuları içerisine rastgele bir veya birkaç adet gps tracker cihazları konulabilir. 
- bu cihazlar sayesinde malzemelerin indirildiği lokasyon kesin olarak görülebilir.
- yağma yapıldığı zaman yağmacıların lokasyonunun bulunmasını sağlayabilir. 

cihazların konumu belirli olduğu için, belli sürelerde tekrar toplanıp kullanılabilir, veya üzerinde yazan notta belirli yetkililere veya tır şöförlerine geri verilmesi sağlanabilir.

**Comment by ramazansancar** [comment]
Merhaba arkadaşlar,
Öncelikle fikriniz için teşekkürler. Bu talebiniz Legal ekibin kontrolü sonrasında reddedildi.
